File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int[] list = new int[9];
        for(int i = 0; i < 9; i++)
        {
            list[i] = scanner.nextInt();
        }
        int max = list[0];
        int location = 1;
        
        for(int i = 0; i < 9; i++)
        {
            if(list[i] > max)
            {
                max = list[i];
                location = i + 1;
            }
        }
        System.out.println(max);
        System.out.println(location);
        scanner.close();
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <stdio.h>

int main()
{
    int list[9];
    int max, location = 0;

    for(int i = 0; i < 9; i++)
    {
        scanf("%d", &list[i]);
    }

    max = list[0]; 
    location = 1;

    for(int i = 1; i < 9; i++)
    {
        if(list[i] > max)
        {
            max = list[i];
            location = i + 1; 
        }
    }

    printf("%d\n%d", max, location);
    return 0;
}